### ISSUE TITLE
'guest ok = no' not working?

### ISSUE THREAD

**Issue by Andy2244** [cifsd-team/cifsd-tools#82]
I can access this share just fine without having a cifsdpwd.db?
While this maps "bad users" to guests, it seems `guest ok = no` is not working?
```
[global]
	netbios name = router
	server string = Cifsd on OpenWrt
	workgroup = WORKGROUP
	interfaces = br-lan 
	bind interfaces only = yes
	ipc timeout = 20
	deadtime = 15
	map to guest = Bad User

[test]
	path = /mnt/sda
	create mask = 0666
	directory mask = 0777
	read only = yes
	guest ok = no
```

**Comment by namjaejeon** [comment]
Thanks for report! Checking.

**Comment by namjaejeon** [comment]
FIXED, please check v2.0.3 version.

**Comment by Andy2244** [comment]
thanks works now!
